packages/halo-theme/src/custom-elements/ef-calendar.less

```diff
@@ -155,6 +155,10 @@
   [part~='selection'] {
     outline: none;
     border: 1px solid @calendar-cell-background-color;
+
+    ::slotted(*) {
+      position: absolute;
```

Comment:
seems like `position: absolute;` display the cell border for halo-theme and solar-theme are not the same.
is this expected result?
<img width="331" alt="Screenshot 2566-09-01 at 13 59 19" src="https://github.com/Refinitiv/refinitiv-ui/assets/102957966/b80b8d4b-f1dd-4917-8338-5369b55aff0b">
<img width="331" alt="Screenshot 2566-09-01 at 13 59 36" src="https://github.com/Refinitiv/refinitiv-ui/assets/102957966/e9960c37-afc5-47cf-bd80-0d3cc6fb0935">

Reply:
Absolutely yes. You can zoom in to look at the border of cell in solar-theme. Then try to press on the cell, you will see that border of cell is overlap on each other.

Reply:
Okay, how about move to elemental-theme instead.